test/util/malicious/out_of_order_builder.go

```diff
@@ -72,14 +72,14 @@ func OutOfOrderExport(b *square.Builder) (square.Square, error) {
 	// sort the blobs in order of namespace. We use slice stable here to respect the
 	// order of multiple blobs within a namespace as per the priority of the PFB
 	sort.SliceStable(b.Blobs, func(i, j int) bool {
-		return bytes.Compare(square.FullNamespace(b.Blobs[i].Blob), square.FullNamespace(b.Blobs[j].Blob)) < 0
+		return bytes.Compare(b.Blobs[i].Blob.Namespace(), b.Blobs[j].Blob.Namespace()) < 0
```

Comment:
LGTM! what is the motivation behind this change? could not find it in the linked issue. Thanks.